src/datepicker/utils.es

```diff
@@ -16,3 +16,33 @@ export const getDate = (dateOrIso) => {
   if (dateOrIso instanceof Date) return dateOrIso;
   return parseISO(dateOrIso);
 };
+
+const stripPrefix = (key, prefix) => {
+  const strippedKey = key.substring(prefix.length);
+  return strippedKey.charAt(0).toLowerCase() + strippedKey.slice(1);
```

Comment:
`key.replace(new RegExp(`^${prefix}`), '')` ?
precompile regex out of the loop and you won't need the function?

Reply:
Regex may even be slower and I would still need to convert first character to lower case

calendarRoot -> root
calendarSuperPicker -> superPicker